pycbc/strain.py

```diff
@@ -354,8 +381,11 @@ def from_cli(opt, dyn_range_fac=1, precision='single'):
                              pd_taper_window) )
         gate_data(strain, gate_params)
 
-
+    strain.injcutter = injcutter
```

Comment:
@tdent yes, a less jargony name would be better. ``injection-reduced-bank`` sounds good to me, as it's more descriptive.